shiny/ui/_utils.py

```diff
@@ -130,3 +131,8 @@ def extract_js_keys(options: Dict[str, Any], parent_key: str = "") -> List[str]:
             value = cast(Dict[str, Any], value)
             js_html_keys.extend(extract_js_keys(value, full_key))
     return js_html_keys
+
+
+def path_pkg_www(*args: str) -> str:
+    base = os.path.join(os.path.dirname(__file__), "..", "www", "shared")
+    return os.path.join(base, *args)
```

Comment:
IMO this abstraction doesn't seem worth it since it's currently only getting used in one place

Reply:
I mean, I think we should use it in more places but I don't think we should make that change in this PR

Reply:
That said, the assumption that we only need to move up once (via `..`) isn't generally correct. So I'll move this into `_theme.py`

Reply:
Oh wait, no, `path_pkg_www()` is a nice way of _avoiding_ having to manage the number of directories you need to go up from anywhere in the shiny package. I think this is worth keeping and using in more places.